Dockerfile

```diff
@@ -35,6 +35,8 @@ ENV UMASK=022
 ENV DOWNLOAD_DIR /downloads
 ENV STATE_DIR /downloads/.metube
 ENV TEMP_DIR /downloads
+ENV PUBLIC_HOST_URL download/

```

Comment:
No need to set defaults here, since they're already set in `app/main.py`. Here we only override configurations that are specifically set for Docker (and are different in `app/main.py`).